Mono.Cecil.props

```diff
@@ -36,14 +36,15 @@
   </PropertyGroup>
   <PropertyGroup Condition=" $(Configuration.StartsWith('net_3_5')) ">
     <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>
-    <DefineConstants>$(DefineConstants);</DefineConstants>
+    <DefineConstants>$(DefineConstants);NET_3_5;</DefineConstants>
   </PropertyGroup>
   <PropertyGroup Condition=" $(Configuration.StartsWith('net_4_0')) ">
     <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
     <DefineConstants>$(DefineConstants);NET_4_0;</DefineConstants>
   </PropertyGroup>
-  <PropertyGroup Condition=" $(NetStandard) ">
-    <TargetFramework>netstandard1.3</TargetFramework>
+  <PropertyGroup Condition="'$(NetStandard)' == 'true'">
```

Comment:
Can't we use '$(NetStandard)' and '! $(NetStandard)'?

Reply:
Most msbuild targets use ```"'$(Var)' == 'true'"``` to allow the variable to be empty. Empty string is not auto-converted to boolean false.

Reply:
Ok thanks!